corehq/apps/app_manager/models.py

```diff
@@ -818,6 +818,7 @@ class FormBase(DocumentSchema):
     form_links = SchemaListProperty(FormLink)
     schedule_form_id = StringProperty()
     custom_instances = SchemaListProperty(CustomInstance)
+    case_references_data = DictProperty()
```

Comment:
not sure if there are any implications of this moving up to the base class that I should worry about / test. it seemed to be fine everywhere i tried it.